runtime/autoload/remote/host.vim

```diff
@@ -180,7 +178,7 @@ command! UpdateRemotePlugins call s:UpdateRemotePlugins()
 
 
 let s:plugins_for_host = {}
-function! s:PluginsForHost(host)
+function! remote#host#PluginsForHost(host)
```

Comment:
It's to allow the host factory function to access the path and specs of its registered plugins. The Python host factory uses is, but I think that it's especially useful when interfacing with a compiled language without support for metaprogramming.